tests/conftest.py

```diff
@@ -93,3 +94,17 @@ def manage_tm_config_for_testing():
 
     if created_cfg_dir:
         os.rmdir(cfg_dir)
+
+
+@pytest.fixture(autouse=True, scope="session")
+def init_mlflow_tracking():
+    uri = os.environ.get("MLFLOW_TRACKING_SERVER_URI", "http://localhost:8080")
+    mlflow.set_tracking_uri(uri=uri)
+
+    yield
+
+
+@pytest.fixture(scope="session")
+def fxt_mlflow_client():
+    uri = os.environ.get("MLFLOW_TRACKING_SERVER_URI", "http://localhost:8080")
+    return mlflow.MlflowClient(uri)
```

Comment:
Do we need to put this in the global conftest.py?
(This might be running in unit/intg tests, too)

Reply:
@goodsong81 I have the next plan which is related with CI-resource usage monitoring through the logging feature of the mlflow ('duration' attribute of the 'run'). that could be done by adding some 'autouse' fixture to each test type (e.g., unit, intg). so i've added mlflow tracking server init & client fixture at the global scope.